compose/.env.es6

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ES_VERSION=6.8.8
+ES_VERSION=6.8.18
 SW_STORAGE=elasticsearch
 IMAGE_TAG=es6
+SW_VERSION=8.7.0
```

Comment:
`SW_VERSION` set in https://github.com/apache/skywalking-docker/blob/master/Makefile#L58

Reply:
I was using the docker compose files by themselves not using make. But that makes sense (no pun instead)

Reply:
It is better to update doc about this env. We don't recommend hardcode version.

Reply:
we use https://github.com/apache/skywalking-docker/blob/master/8/v8.7.0.sh  to export `SW_VERSION` when we dev and test. and I agree with @wu-sheng , do not harcode here.

Reply:
Makes sense. Have pushed a change removing SW_VERSION